components/director/internal/open_resource_discovery/validator.go

```diff
@@ -52,7 +58,9 @@ func (v *DocumentValidator) Validate(ctx context.Context, documents []*Document,
 
 		if len(currentDocumentErrors) > 0 {
 			log.C(ctx).Infof("There are %d validation errors from API Metadata Validator", len(currentDocumentErrors))
-			deleteInvalidResourcesFromDocument(documents[i], currentDocumentErrors)
+
```

Comment:
Probably should merge Global ignore list and name-spaced ignore list here before proceed with clearing the results.